help/en/html/doc/Technical/StartUpScripts.shtml

```diff
@@ -180,31 +180,6 @@
       copied from the distributed disk image onto the user's disk
       drive.</p>
 
-      <h3>Bundle Details</h3>JMRI has registered five application
-      signatures:
-
-      <dl>
-        <dt>JMRD</dt>
```

Comment:
Not sure whether those registered IDs are relevant or not.  JMRD still appears as CFBundleSignature in scripts/MacOSXapps/TemplateApp.app/Contents/Info.plist and hence in the distributed macOS app bundle's Contents/Info.plist; JMRP doesn't seem to be anywhere.

Reply:
The CFBundleSignature key is not listed in [Core Foundation Keys](https://developer.apple.com/library/content/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html#//apple_ref/doc/uid/20001431-102070) and [this SO answer](https://stackoverflow.com/a/1898662/176160) suggests the CFBundleSignature is only used for compatibility with MacOS Classic apps. We can probably remove it from the bundle (I can't find the registry of IDs anymore either).